src/index.ts

```diff
@@ -1,19 +1,47 @@
-const allInvoices = [];
-const invoiceForm = document.getElementById('form');
+const allInvoices: object[] = [];
+const invoiceForm: HTMLElement | null = document.getElementById('form');
 const items = document.getElementsByClassName('item');
 const addProductField = document.getElementById('add product field');
 const invoicesList = document.getElementById('invoices list');
 
-class Invoice {
-  constructor(items, tax, name) {
-    this.items = items;
-    this.name = name;
-    // this.totalWithoutTax = +this.items
-    //   .reduce((prev, cur) => cur.total + prev, 0)
-    //   .toFixed(2);
-
-    //краще виконувати певні операції прямо в конструткорі як тут або виносити їх в окрему функцію, як на 27 рядку(totalWithoutTax())?
+type newInvoice = {
+  position?: string;
+  companyName?: string;
+  type: string;
+  items: {
+    name: string;
+    total: number;
+    price: number;
+    quantity: number;
+  }[];
+  totalWithTax: number;
+  totalWithoutTax: Function;
+  vat: number;
+  id: number;
+  getDate: Function;
+  name: string;
+};
+
+type item = {
```

Comment:
```suggestion
type Item = {
```
for types and interfaces use PascalCase